settings.gradle

```diff
@@ -19,8 +19,26 @@ buildscript {
     dependencies {
         classpath 'org.yaml:snakeyaml:2.2'
     }
-    repositories {
-        mavenCentral()
+
+    if (!gradle.root.hasProperty("mozconfig")) {
+        // in app-services
+        repositories {
+            mavenCentral()
+        }
+    } else {
+        // big copy/paste from mobile/android/shared-settings.gradle.
+        gradle.ext.mozconfig = gradle.root.mozconfig
+
+        repositories {
+            gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
+                maven {
+                    url repository
```

Comment:
One minor quibble - can you please use `url = repository` for this? Gradle is starting to complain about not doing so being deprecated and this will save a couple places having to change it later.